ci/incremental/setup_conda_environment.sh

```diff
@@ -5,23 +5,20 @@ set -v -e
 CONDA_INSTALL="conda install -q -y"
 PIP_INSTALL="pip install -q"
 
+
 # Deactivate any environment
 source deactivate
 # Display root environment (for debugging)
 conda list
 # Clean up any left-over from a previous build
 # (note workaround for https://github.com/conda/conda/issues/2679:
 #  `conda env remove` issue)
-conda remove --all -q -y -n $CONDA_ENV
+conda remove --all -q -y -n pandas-dev
 
 echo
 echo "[create env]"
-time conda env create -q -n "${CONDA_ENV}" --file="${ENV_FILE}" || exit 1
+time conda env create -q -n pandas-dev --file="${ENV_FILE}" || exit 1
 
-# Activate first
-set +v
-source activate $CONDA_ENV
```

Comment:
This one can't be removed, as the rest of the next step is to remove pandas from the environment, so it needs to be active